src/asmcnc/apps/drywall_cutter_app/screen_drywall_cutter.py

```diff
@@ -126,6 +129,14 @@ def __init__(self, **kwargs):
     def home(self):
         self.m.request_homing_procedure('drywall_cutter', 'drywall_cutter')
 
+    def select_tool(self):
+        selected_tool_name = self.tool_selection.text
+
+        for tool in self.tool_options:
+            if tool[0] == selected_tool_name:
+                self.dwt_config.load_cutter(tool[1])
+                break
```

Comment:
Store cutters in a dictionary instead of a list: then, instead of needing to loop through a list to match the tool selection, can just use the tool name as a key. This is less likely to cause lag in the long run, especially if we then expand cutter options/use this logic elsewhere.